macaw-aarch32-syntax/test-data/get-regs.cbl

```diff
@@ -0,0 +1,54 @@
+(defun @id ((regs AArch32Regs)) AArch32Regs
+  (start start:
+    (let r0 (get-reg r0 regs))
```

Comment:
Is it worth including corresponding `set-reg` statements for each register as well?